js/apps/account-ui/src/api/methods.ts

```diff
@@ -109,8 +109,24 @@ export async function getCredentials({ signal, context }: CallOptions) {
   return parseResponse<CredentialContainer[]>(response);
 }
 
-export async function getLinkedAccounts({ signal, context }: CallOptions) {
-  const response = await request("/linked-accounts", context, { signal });
+export type LinkedAccountQueryParams = {
+  first: number;
+  max: number;
+  search?: string;
+  linked?: boolean;
+};
```

Comment:
There is already a type for the pagination present that can be re-used here.

```suggestion
export type LinkedAccountQueryParams = PaginationParams & {
  search?: string;
  linked?: boolean;
};
```